content/2016.09.jedit-trackers-with-kinto-en.md

```diff
@@ -0,0 +1,403 @@
+Title: Getting jEdit Trackers off the Ground using Kinto and Kinto.js
+Date: 2016-09-09
+Lang: en
+Url: jedit-trackers-with-kinto
+Slug: jedit-trackers-with-kinto
+Summary: My experience using Kinto and Kinto.js to browse jEdit tickets offline
+
+# Getting jEdit Trackers off the Ground using Kinto and Kinto.js
+
+This is a guest post by [Eric Le Lay](https://github.com/elelay) on his experience using Kinto.
+
+## Introduction
+
+Do you know [jEdit](http://jedit.org) — *programmer's text editor*?  
+It's a multi-platform text editor, written in java, with a myriad of plugins.
+I've been part of the development team for ten years.
+It's hosted on Sourceforge, with the code base under *subversion*.
+
+This is a labour of love for us, contributing during our free time.
+I don't always have an Internet access on holidays. Or it's via a short Ethernet cable in a dark and overheated room.
+Don't lock me out of the sunny garden! Give me everything I need off-line!
```

Comment:
I feel like the sentences in this paragraph do not have smooth transitions between them. Specifically, there's a missing link between the ideas that you only work on jEdit during your free time, and that you mostly have free time when you're on holidays. The reader has to infer that. By contrast, the French version has phrases like `Pour ma part` and `Tout ça pour dire que`. How about putting something like `For me, that's when I'm on holidays.` between the first and second sentences? Then, you could change the second sentence to something like `I don't always have Internet access where I'm staying`, or `on the road`, or `when I'm abroad`. [N.B. "access" isn't countable so you don't say "an access".]

I also think there's a missing link between the second and third sentences, but I like the phrase "Don't lock me out", so I'd say keep that.